assets/css/_circle_button.scss

```diff
@@ -0,0 +1,27 @@
+.c-circle-button {
+  width: 2rem;
+  height: 2rem;
+
+  background-color: $color-gray-50;
+  border-radius: 50%;
+
+  &:hover {
+    background-color: $color-gray-100;
+  }
+
+  &:focus-within {
+    border: 0.0625rem solid $color-eggplant-500;
```

Comment:
I could be wrong, but I think this is applying the focus border when you click on the button, which ... I don't _think_ that's what focus state is supposed to do? I think the behavior is supposed to be that we get only the "active" state (the darker circle around the avatar) on click, not the focus outline as well.

Reply:
The thing is that the active state is actually orthogonal from the hover / focus states, and it just so happens that clicking on the button both focuses it and toggles the active state. I've asked design how they want the precedence to work when we're in both states, but for now I'm letting the interaction states override the active / inactive state since the other way around seems like it would be worse for usability.

Reply:
Changed to use `:focus-visible` so the focus state only appears when tabbing.